hokusai/production.yml

```diff
@@ -199,7 +216,7 @@ kind: CronJob
 metadata:
   name: kaws-update-site-map
 spec:
-  schedule: 7 15 * * *
+  schedule: 7 16 * * *
```

Comment:
Unrelated change (sorry): I noticed that Kaws schedules 2 different crons for the same moment.